app/controllers/inventories_controller.rb

```diff
@@ -2,5 +2,37 @@
 
 # InventoriesController
 class InventoriesController < ApplicationController
-  def index; end
+  def index
+    @user = current_user
+    @inventories = @user.inventories unless @user.nil?
+  end
+
+  def show; end
+
+  def destroy
+    @inventory = Inventory.find(params[:id])
```

Comment:
[ Optional] I think it's better to use `find_by_id(params[:id])` as this will return a nil object instead of crashing the app if the ID does not exist